### PR TITLE
chore(svg-icon): add ExpandMore, StarCircle, Remove, ArrowDownward

### PR DESCRIPTION
**Pull request by dierat**
### Summary:
We're migrating the `SvgIcon` usage in `traject` to use the EDS version of the components. First we have to deal with the 5 icons that are being used in `traject` but don't exist in `EDS`. 1/5 will stay in `traject` (happening in https://github.com/FB-PLP/traject/pull/8848), and the other 4/5 will be copied into this repo (happening in this PR).

Please note that the actual paths are different for 3/4 because I found or made rounded versions of them to be consistent with the rest of the icon set. 2/4 are not in the figma files; I'll make a ticket for design to add them when they have time.

### Screenshots
All icons story, before
All icons story, before this change

All icons story, after
All icons story, after this change

### Test Plan:
`yarn start`,
navigate to `/?path=/story/svgicon--all-icons`,
verify the ExpandMore, StarCircle, Remove, ArrowDownward are now in the list,
and verify their size and roundededness matches the other icons.

### CODE REVIEW

File: packages/components/src/Icons/custom/RemoveRounded.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M3.75 12.25C3.75 12.8 4.2 13.25 4.75 13.25H18.75C19.3 13.25 19.75 12.8 19.75 12.25C19.75 11.7 19.3 11.25 18.75 11.25H4.75C4.2 11.25 3.75 11.7 3.75 12.25Z" />,
```
Comment (dierat) [inline packages/components/src/Icons/custom/RemoveRounded.tsx:5]: I made this one, so this path doesn't exist in `traject` or in the UI kit. Here's the blocky one in traject: https://github.com/FB-PLP/traject/blob/77d303699345622a3f8e0aebe013fbf3c1a7d421/app/assets/javascripts/v2/core/EDSCandidates/SvgIcon/Remove.jsx

File: packages/components/src/Icons/custom/ExpandMoreRounded.tsx [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M15.88 9.29001L12 13.17L8.11999 9.29001C7.72999 8.90001 7.09999 8.90001 6.70999 9.29001C6.31999 9.68001 6.31999 10.31 6.70999 10.7L11.3 15.29C11.69 15.68 12.32 15.68 12.71 15.29L17.3 10.7C17.69 10.31 17.69 9.68001 17.3 9.29001C16.91 8.91001 16.27 8.90001 15.88 9.29001V9.29001Z" />,
```
Comment (dierat) [inline packages/components/src/Icons/custom/ExpandMoreRounded.tsx:5]: I got this path from the UI kit: https://www.figma.com/file/YPTOEVRuas5ytXQEcwm4FK/UI-Kit-Sticker-Sheet-Core?node-id=649%3A2126
Here's the blocky version in `traject`: https://github.com/FB-PLP/traject/blob/77d303699345622a3f8e0aebe013fbf3c1a7d421/app/assets/javascripts/v2/core/EDSCandidates/SvgIcon/ExpandMore.jsx

File: packages/components/src/Icons/custom/ArrowDownwardRounded.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M11 5V16.17L6.11999 11.29C5.72999 10.9 5.08999 10.9 4.69999 11.29C4.30999 11.68 4.30999 12.31 4.69999 12.7L11.29 19.29C11.68 19.68 12.31 19.68 12.7 19.29L19.29 12.7C19.68 12.31 19.68 11.68 19.29 11.29C18.9 10.9 18.27 10.9 17.88 11.29L13 16.17V5C13 4.45 12.55 4 12 4C11.45 4 11 4.45 11 5Z" />,
```
Comment (dierat) [inline packages/components/src/Icons/custom/ArrowDownwardRounded.tsx:5]: I got this path from the UI kit: https://www.figma.com/file/YPTOEVRuas5ytXQEcwm4FK/UI-Kit-Sticker-Sheet-Core?node-id=649%3A2126

Here's the blocky version in `traject`: https://github.com/FB-PLP/traject/blob/77d303699345622a3f8e0aebe013fbf3c1a7d421/app/assets/javascripts/v2/core/EDSCandidates/SvgIcon/ArrowDownward.jsx

Comment (anniehu4) [reply packages/components/src/Icons/custom/ArrowDownwardRounded.tsx:5]: ohh shoot, sorry I didn't clarify this more. Material UI published component versions of these icons that we can use -- I added all of the rounded ones in https://github.com/chanzuckerberg/edu-design-system/releases/tag/all-rounded-icons, and then only committed the icons in the UI kit at that time.

For consistency, we should do
```
git fetch --tags
git co tags/all-rounded-icons -- packages/components/src/Icons/<IconName>.tsx
```

Then any icon that wasn't published by MUI should go within this `custom/` folder (I think only `StarCircleRounded` isn't there)

File: packages/components/src/Icons/custom/StarCircleRounded.tsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import createSvgIcon from "../../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <path d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z" />,
```
[the remark this answers is not in the document]
Comment (anniehu4) [reply packages/components/src/Icons/custom/StarCircleRounded.tsx:5]: this looks really similar to `StarsRounded` from [material ui](https://mui.com/components/material-icons/)
Screen Shot 2021-10-13 at 6 07 51 PM

I'd be down to checkout that file and then rename it to StarCircle (why is it plural stars lol)

**Pull request merged**